cmd/common.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+const defaultMaxTries = 3
```

Comment:
this is not common to `cmd`. this is common to `ytsync`. you also don't need a separate `common.go` file. just put it in an existing file